internal/cli/root_test.go

```
@@ -794,6 +794,47 @@ func TestVersion(t *testing.T) {
+	home := t.TempDir()
+	t.Setenv("HOME", home)
+	if exitCode := Execute(t.Context(), []string{"init"}, strings.NewReader(""), &bytes.Buffer{}, &bytes.Buffer{}, "test"); exitCode != 0 {
+		t.Fatalf("init exit code = %d", exitCode)
+	}
+	workerPath := filepath.Join(home, ".machinist", "worker.toml")
+	file, err := os.OpenFile(workerPath, os.O_APPEND|os.O_WRONLY, 0)
+	if err != nil {
+		t.Fatal(err)
+	}
+	if _, err := file.WriteString("\n[repositories.example]\npath = \"" + filepath.ToSlash(t.TempDir()) + "\"\n"); err != nil {
+		file.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `file.Close` is not checked

(errcheck)